doc/SLSKPROTOCOL.md

```diff
@@ -231,64 +235,53 @@ and callbacks for the messages are set in pynicotine.py.
-| Human       | 160         | 32          | d51c9a7e9353746a6020f9602d452929                                                                | 1             |
-| Hex         | a0 00 00 00 | 20 00 00 00 | 64 35 31 63 39 61 37 65 39 33 35 33 37 34 36 61 36 30 32 30 66 39 36 30 32 64 34 35 32 39 32 39 | 01 00 00 00   |
+*... continued:*
+| *Data*    | Version     | Length      | Hash                                                                                            | Minor Version |
+| --------- | ----------- | ----------- | ----------------------------------------------------------------------------------------------- | ------------- |
+| **Human** | 160         | 32          | d51c9a7e9353746a6020f9602d452929                                                                | 1             |
+| **Hex**   | a0 00 00 00 | 20 00 00 00 | 64 35 31 63 39 61 37 65 39 33 35 33 37 34 36 61 36 30 32 30 66 39 36 30 32 64 34 35 32 39 32 39 | 01 00 00 00   |
 
-*Message as a Hex Stream* **48 00 00 00 01 00 00 00 08 00 00 00 75 73 65
-72 6e 61 6d 65 08 00 00 00 70 61 73 73 77 6f 72 64 a0 00 00 00 20 00 00
-00 64 35 31 63 39 61 37 65 39 33 35 33 37 34 36 61 36 30 32 30 66 39 36
-30 32 64 34 35 32 39 32 39 01 00 00 00**
+*Message as a Hex Stream:*
+  - **48 00 00 00 01 00 00 00 08 00 00 00 75 73 65 72 6e 61 6d 65 08 00 00 00 70 61 73 73 77 6f 72 64 a0 00 00 00 20 00
+      00 00 64 35 31 63 39 61 37 65 39 33 35 33 37 34 36 61 36 30 32 30 66 39 36 30 32 64 34 35 32 39 32 39 01 00 00 00**
 
-### Data Order
-
-  - Send
+*Data Order:*
+  - Send Login Attempt
     1.  **string** <ins>username</ins>
-    2.  **string** <ins>password</ins> **A non-empty string is required**
-    3.  **uint32** <ins>version number</ins> *160* for Nicotine+
-    4.  **string** <ins>MD5 hex digest of concatenated username & password</ins>
-    5.  **uint32** <ins>minor version</ins> Minor version (0x13000000 for 157 ns 13e, 0x11000000 for 157 ns 13c)
+    2.  **string** <ins>password</ins> *A non-empty string is required*
+    3.  **uint32** <ins>version number</ins> **160** for Nicotine+
+    4.  **string** <ins>hash</ins> *MD5 hex digest of concatenated username & password*
+    5.  **uint32** <ins>minor version</ins> **0x13000000** for 157 ns 13e, **0x11000000** for 157 ns 13c
   - Receive Login Success
-    1.  **bool** <ins>success</ins> 1
-    2.  **string** <ins>greet</ins> A MOTD string
+    1.  **bool** <ins>success</ins> **1**
+    2.  **string** <ins>greet</ins> *MOTD string*
     3.  **uint32** <ins>Your IP Address</ins>
-    4.  **string** <ins>MD5 hex digest of the password string</ins> *Windows Soulseek uses this hash to determine if it's connected to the official server*
+    4.  **string** <ins>hash</ins> *MD5 hex digest of the password string (Windows SoulseekQt uses this hash to determine if it's connected to the official server)*
```

Comment:
This statement is probably not relevant anymore, and should be removed. Windows Soulseek refers to Soulseek NS before the new server was introduced.